packages/@okta/vuepress-site/docs/guides/implement-saml2/setup-app/index.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Set up your application
+---
+
+Create the client application that you want to use:
+
+1. From the Admin Console side navigation, select **Applications** and then **Applications**.
+
+2. On the Applications page, click **Add Application** and then **Create New App**.
+
+3. Select **Native** as the type of app **Platform** on the **Create a New Application Integration** window and then click **Create**.
```

Comment:
```suggestion
3. Select **Native** as the type of app Platform on the **Create a New Application Integration** window and then click **Create**.
```

Reply:
@jakubvul Platform is the name of the field in the dialog box. Do we not bold names of fields?